index.d.ts

```diff
@@ -5,6 +5,11 @@ declare namespace waitForLocalhost {
 		 */
 		port?: number;
 
+		/**
+		@default '/''
+		 */
```

Comment:
Needs a doc description. Same with the readme.

Reply:
✔️